### DISCUSSION TITLE
Error creating new project

### DISCUSSION THREAD

**Discussion by razorbohan** [WalletConnect/walletconnect-monorepo#1878] [Developer Support]
Have just created an account in the cloud and trying to create a new project, but receiving this error after inputing project name:

<img width="610" alt="Снимок экрана 2023-01-19 в 17 02 12" src="https://user-images.githubusercontent.com/31306103/213418208-dde7e2ed-d2dd-42bb-aab6-29931e07de67.png">

**Comment by crypblizz8** [comment]
Thanks for sharing this @razorbohan 

So the steps you took were:
1. Create new email account via SignUp
2. Login
3. Create a new Project
4. In the dialog, it asked for a name for the project

I tried to reproduce your error with both Sign in with Ethereum/WalletConnect + email but wasn't able to. 

Could you share more context? Thanks

**Reply by razorbohan** [reply]
Yes, those are exactly the steps I took. Please see screen recording:

https://user-images.githubusercontent.com/31306103/213431569-2b132025-f49c-4e64-a8d6-52d754bdec8c.mov
